core/tests/README.md

```diff
@@ -0,0 +1 @@
+The root-files-config.json holds the root files that are valid, there are two places a root file can go, the RUN_NO_TESTS_ROOT_FILES and RUN_ALL_TESTS_ROOT_FILES. If the CI gives you an error regarding this config file, please add the root file to one of these lists, depending on if it needs to run tests or not. For more information, refer to the wiki here: https://github.com/oppia/oppia/wiki/Partial-CI-Tests-Structure#root-files-config
```

Comment:
It should be clear from reading this what a root file is. I know you document it in the wiki, but this readme should be understandable on its own.